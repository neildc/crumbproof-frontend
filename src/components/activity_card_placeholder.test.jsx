import React from 'react';
import { shallow } from 'enzyme';
import ActivityCardPlaceholder from './activity_card_placeholder';

it('renders without throwing', () => {
  const component = shallow(<ActivityCardPlaceholder />);

  expect(component).toMatchSnapshot();
});
