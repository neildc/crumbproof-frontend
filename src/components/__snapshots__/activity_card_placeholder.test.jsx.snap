// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without throwing 1`] = `
<Card
  expandable={false}
  expanded={null}
  initiallyExpanded={false}
  style={
    Object {
      "height": "800px",
    }
  }
>
  <div
    className="placeholder-wrapper"
  >
    <div
      className="placeholder-item"
    >
      <div
        className="animated-background"
      >
        <div
          className="background-masker content-top"
        />
        <div
          className="background-masker content-first-end"
        />
        <div
          className="background-masker content-second-line"
        />
        <div
          className="background-masker content-second-end"
        />
        <div
          className="background-masker content-third-line"
        />
        <div
          className="background-masker content-third-end"
        />
      </div>
    </div>
  </div>
</Card>
`;
